.github/workflows/pr-open.yml

```diff
@@ -2,14 +2,16 @@ name: Pull Request
 
 on:
   pull_request:
-
+    branches:
+      - main
```

Comment:
Is the idea to only run PR actions while pointing to main?  That'd make sense, but change our offering a bit.

Reply:
yes that is the idea, since everything goes through main, raising PR to other branches might use for code updates, but again when final PR is made to main branch, CI/CD with sandbox deployment will start.